src/Surfnet/Stepup/Identity/Event/WhitelistCreatedEvent.php

```diff
@@ -19,9 +19,14 @@
 namespace Surfnet\Stepup\Identity\Event;
 
 use Surfnet\Stepup\Identity\Collection\InstitutionCollection;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData\RightToObtainDataInterface;
 
-class WhitelistCreatedEvent implements WhitelistEvent
+class WhitelistCreatedEvent implements WhitelistEvent, RightToObtainDataInterface
 {
+    protected static $whitelist = [
+        'whitelisted_institutions',
```

Comment:
@pmeulen Am I right in the assumtion that the events associated to the whitelist actions are be candidates to skip. They seem irrelevant to the Identity from a data portability perspective. But I might be wrong in that aspect? Whats your take on this?

Reply:
I agree. Events that handle middleware, institution and whitelist configuration do not contain any personal data and must be skipped.